src/main/java/net/sf/jabref/logic/journals/AbbreviationParser.java

```diff
@@ -30,9 +30,12 @@ public void readJournalListFromResource(String resourceFileName) {
     }
 
     public void readJournalListFromFile(File file) throws FileNotFoundException {
-        readJournalList(new FileReader(Objects.requireNonNull(file)));
+        try(FileReader fr = new FileReader(Objects.requireNonNull(file))) {
+            readJournalList(fr);
+        } catch(IOException e) {
+            LOGGER.info("Could not read journal list from file " + file.getAbsolutePath());
+        }
```

Comment:
To keep the semantics of this method the same, I would suggest to remove the catch block as before a FileNotFoundException was thrown when the file was not there. In the current code, a FileNotFoundException` is not thrown as a FileNotFoundException is a subtype of IOException it is caught by the catch block.